consul/consul_test_server.go

```diff
@@ -32,6 +32,32 @@ func CreateTestServer(t *testing.T) *testutil.TestServer {
 	return server
 }
 
+const MasterToken = "masterToken"
+
+func CreateSecuredTestServer(t *testing.T) *testutil.TestServer {
+	ports, err := getPorts(6)
+	assert.NoError(t, err)
+
+	server, err := testutil.NewTestServerConfig(func(c *testutil.TestServerConfig) {
+		c.Datacenter = fmt.Sprint("dc-", time.Now().UnixNano())
+		c.Ports = &testutil.TestPortConfig{
```

Comment:
Can you extract this to a fucntion that returns `*testutil.TestPortConfig`

Reply:
done